x/community/testutil/staking_rewards.go

```diff
@@ -258,10 +258,15 @@ func (suite *stakingRewardsTestSuite) TestStakingRewards() {
 			params.StakingRewardsPerSecond = tc.rewardsPerSecond
 			keeper.SetParams(ctx, params)
 
+			stakingRewardEvents := sdk.Events{}
+
 			for {
 				// run community begin blocker logic
 				suite.testFunc(ctx, keeper)
 
+				// accumulate event rewards from events
+				stakingRewardEvents = append(stakingRewardEvents, filterStakingRewardEvents(ctx.EventManager().Events())...)
```

Comment:
I made this a little cleaner by moving the event accumulation above the break statement to catch all testFunc runs, in addition collected the events so we can add the assertion for no tracking for zero rewards